### PR TITLE
Claim URL + Submit for review

### PR DESCRIPTION
**Pull request by diegoquinteiro**
This PR adds two new methods to the client, supporting 2 new APIs we'll be releasing:

- [x] Review submission API
- [x] Domain claiming API

### CODE REVIEW

File: src/Facebook/InstantArticles/Client/Client.php [hunk 1]
```diff
@@ -242,4 +242,42 @@ public function getArticlesURLs($limit = 10, $development_mode = false)
 
         return $articleURLs;
     }
+
+    /**
+     * Claims an URL for the page
+     *
+     * @param string $url The root URL of the site without the protocol (ex: exemple.com/path)
+     */
+    public function claimURL($url)
```
Comment (everton-rosario) [inline src/Facebook/InstantArticles/Client/Client.php]: What if informing $url with protocols?
Shouldn't be treated/rejected in code?

**Comment by everton-rosario** [comment]
Add a test case and treatment when invalid $url is informed.
Like:
* http://example.com/path
* ftp://example.com/path
* invalidcontent/path

**Comment by diegoquinteiro** [comment]
@everton-rosario I've fixed the http:// and https:// cases;

The ftp:// and invalidcontent cases will be rejected by the server.

**Comment by everton-rosario** [comment]
:shipit:

**Pull request merged**